show/main.py

```diff
@@ -1384,7 +1384,12 @@ def runningconfiguration():
 def all(verbose):
     """Show full running configuration"""
     cmd = "sonic-cfggen -d --print-data"
-    run_command(cmd, display_cmd=verbose)
+    output = json.loads(run_command(cmd, display_cmd=verbose, return_cmd=True))
+    if not multi_asic.is_multi_asic():
+        bgpraw_cmd = "sudo {} -c 'show running-config'".format(constants.RVTYSH_COMMAND)
+        bgpraw = run_command(bgpraw_cmd, display_cmd=verbose, return_cmd=True)
```

Comment:
if the bgp container is down, bgpraw will contain error messge. Do we want to store that or make `output['bgpraw'] `empty?

Reply:
Empty running-config value for bgp down is a good suggestion.

Reply:
The run_command was calling subprocess. The output will be empty though subprocess calling is error. 
So, the `bgpraw` will be empty if bgp container is down.